### ISSUE TITLE
C# missing from language list when extension is installed

### ISSUE THREAD

**Issue by kubal5003** [microsoft/vscode#90128]
.cs files appear as Plain Text and when I try to change that it looks like the C# language is missing from the list.

I have tried installing and uninstalling the C# extension, also tried different versions, but nonetheless I'm not able to get .cs file support.

![image](https://user-images.githubusercontent.com/1132035/73929525-3739f300-48d5-11ea-81ab-617b0c9210d6.png)

VS Code details:
```
Version: 1.41.1
Commit: 26076a4de974ead31f97692a0d32f90d735645c0
Date: 2019-12-18T14:57:51.166Z
Electron: 6.1.5
Chrome: 76.0.3809.146
Node.js: 12.4.0
V8: 7.6.303.31-electron.0
OS: Darwin x64 18.7.0
```
C# plugin version: 1.21.10 (also tried a couple other versions)

**Comment by eamodio** [comment]
@kubal5003 I can't reproduce this -- do you have any custom settings changing this or extensions that may be conflicting?

**Comment by kubal5003** [comment]
I think I figured what was wrong. The syntax highlight is actually part of "C# Language Basics" extension that I had disabled. (I remember disabling some extensions I wasn't using for performance/memory reasons a couple months ago)

It wasn't immediately obvious that C# support is spread across two extensions and one of them is built into VS Code. What is even weirder is that now that I enabled that extension and did restart VS Code it's not displayed on the list of enabled extensions:
![image](https://user-images.githubusercontent.com/1132035/73979843-42ba0800-492f-11ea-9265-11e8c744dbbb.png)

It does not appear on the list of Disabled either. Not sure if it just disappeared?
